Makefile

```diff
@@ -14,10 +14,11 @@ clean:
 # Process .ego templates (skippable)
 EGOS := $(shell find . -name *.ego | sed -e s/.ego/.ego.go/)
 $(EGOS):
+	go get github.com/benbjohnson/ego/cmd/ego
 	ego api/views
 
 # Fetch dependencies
-vendor: glide.yaml
+init: glide.yaml $(EGOS)
```

Comment:
i think this needs to be specifically `vendor: glide.yaml` for make's caching to work. if you declare the `init` rule separately, we can have the best of both worlds.

Reply:
hmm I see. so how about:

```
init: $(EGOS) vendor

# Fetch dependencies
vendor: glide.yaml
	glide install
	go install
```

Reply:
looks good! i'll try it out when the branch is ready.